### DISCUSSION TITLE
HLS not work: destroyed (no one is publishing to path 'live')

### DISCUSSION THREAD

**Discussion by liuyl1992** [bluenviron/mediamtx#978] [Questions]
![image](https://user-images.githubusercontent.com/16396828/172534013-4a0e06d7-93b9-424d-9b79-07bc8c7f756f.png)
![image](https://user-images.githubusercontent.com/16396828/172534062-7a301c6d-9c4d-4c6c-be32-168d28669888.png)
![image](https://user-images.githubusercontent.com/16396828/172534101-a0e1fa0f-f441-4ff7-bd80-8f4567ec2604.png)
Why is this happening and how can it be fixed

**Comment by aler9** [comment]
Hello, you're trying to publish a stream with HLS, and this is currently unsupported as clearly stated in the matrix in the README, but there's no need for that, you can publish a stream with RTSP and read it with HLS:

You can publish with RTSP with:

```
ffmpeg -re -i rtsp://camera-url -c copy -f rtsp rtsp://localhost:8554/live
```

And read with HLS:

```
http://localhost:8888/live
```

[a comment by an automated account is not shown]
